### DISCUSSION TITLE
SOLVED: Internal links not working properly

### DISCUSSION THREAD

**Discussion by ninetailedmachine** [community/community#23093] [Other Feature Feedback, Questions, & Ideas]
<p>So I am kinda new to html and github. I have my github pages website up and running, but when you click on a link in the nav bar, it shows a 404 error because the link repeats itself. Example: Normal url upon loading up the site is <a href="http://example.github.io/example.html/" class="inline-onebox-loading" rel="nofollow noopener">example.github.io/example.html/</a>. When you click on a link in the nav bar it goes to <a href="http://example.github.io/example.html/example.html" class="inline-onebox-loading" rel="nofollow noopener">example.github.io/example.html/example.html</a>, and that shows a 404 error. </p>
<p>Link to main repository --&gt; <a href="https://github.com/ninetailedmachine/ninetailedmachine.github.io" rel="nofollow noopener">https://github.com/ninetailedmachine/ninetailedmachine.github.io</a></p>
<p>EDIT: I’ve solved the problem</p>

**Comment by AndreaGriffiths11** [comment]
<p><a class="mention" href="/ninetailedmachine">@ninetailedmachine</a> Thanks for the update! Would you mind sharing your solution?</p>

**Comment by ninetailedmachine** [comment, marked as answer]
<p>I had made separate repositories instead of making a folder inside of the main repository. So I made a folder in the main repo, changed the html a bit to follow that path, deleted the other repositories and it worked! Obviously, I’m new to github, so this took me longer to figure out.</p>

**Reply by sabbir292** [reply]
just remove the forward slash infrom of the recipes folder when linking 

ex: <a href = "recipes/recipeName.html>recipeName</a> it will work , no need to add js

**Comment by AndreaGriffiths11** [comment]
<p><span class="mention">@ninetailedmachineThanks</span> a bunch for coming back to share! We’re looking forward to seeing you around!</p>

**Comment by six6out** [comment]
<p>You can also use JS in the end to force it to go to that link.</p>
<p>&lt;a href=“example.html” onclick=“window.open(‘example.html’, ‘_self’);”&gt;</p>
<p>  Example</p>
<p>&lt;/a&gt;</p>

**Comment by mzaifquraishi** [comment]
<p>Hi How do you solve the problem can you tell me some details.</p>
<p>do i have to create a forlder within repo and upload files into that folder?</p>
<p>My Repo-&gt; <a href="https://github.com/mzaifquraishi/mzaifquraishi.github.io" rel="nofollow noopener">https://github.com/mzaifquraishi/mzaifquraishi.github.io</a> </p>

**Comment by six6out** [comment]
<p><strong>Your website looks great so far!</strong></p>
<p>GitHub Links don’t work unless forced via JS.</p>
<pre><code>&lt;a href="example.html" onclick="window.open('example.html', '_self');"&gt;

  Example

&lt;/a&gt;
</code></pre>
<p>Change example.html to your page and change the Example text to your link text.</p>

**Comment by moinmk** [comment]
<p>thanks that is exacatly what i was looking for.</p>

**Comment by Codefitness21** [comment]
<p>How could I use your example above and apply it to this? : </p>
<p>&lt;a href=“Services.html”&gt;Services&lt;/a&gt;</p>

**Comment by Gikunjuu** [comment]
<p>Thank You so much. However, can you provide more insight on how you would write this<br>
&lt;a href="…/projects/shoegods/shoegods.html" onclick=“window.open(’…/projects/shoegods/shoegods.html’, ‘_self’);“class=“project” aria-label=“Shoegods” name=“Shoegods” data-scroll data-scroll-repeat data-scroll-offset=“50%, 50%” data-scroll-call=“changeBG” data-background=”<span class="hashtag">#D0503E</span>” data-next-color=“DFD3C3” style=“color: <span class="hashtag">#DFD3C3</span>”&gt;</p>

**Comment by Ashutosh002** [comment]
<p>Thank you, it helped!</p>

**Comment by jAmEsNico7** [comment]
<p>Was having trouble with the same problem earlier but this comment really saved me, thanks :heart:</p>

**Comment by MdHasanRaju** [comment]
<p>thanks a lot. could you please tell me why am i getting same error if i try to to navigate the first html link? how can i fix this problem?</p>

**Reply by trishlakhandelwal** [reply]
Facing the same problem. did u find the solution?

**Comment by makhon3g** [comment]
https://makhon3g.github.io/seo4/
This is my live link but does not work properly what I will do?

**Comment by p3g-18** [comment]
whenever, I open my project link it gets me back to my GitHub homepage instead of the project link.

**Reply by p3g-18** [reply]
does anyone know any solution.

**Reply by relicote** [reply]
Another member help me with the same question in this discussion. 

just remove the forward slash infrom of the recipes folder when linking

ex: <a href = "recipes/recipeName.html>recipeName it will work , no need to add js
